File: 6th Semester/Programming Paradigms/NP2 Exercises - Part 2/ex11.py
# – Escreva um programa que pergunte o deposito inicial e a taxa de juros de uma poupança. Exiba os
# valores mês a mês para os 24 primeiros meses. Escreva o total ganho com juros no período.
meses = ["Janeiro", "Fevereiro", "Março", "Abril", "Maio", "Junho", "Julho", "Agosto", "Setembro", "Outubro", "Novembro", "Dezembro"]
montante = float(input('Digite o depósito inicial: '))
taxaJuros = float(input('Agora informa a taxa de juros (%): '))
taxaJuros /= 100
for i in range(24):
    montante *= (1+taxaJuros)
    print(f'{meses[i%12]} ano {i//12+1}: R${montante:.2f}')

File: 5th Semester/Programming Paradigms/NP2 Exercises - Part 2/ex11.c
// – Escreva um programa que pergunte o deposito inicial e a taxa de juros de uma poupança. Exiba os
// valores mês a mês para os 24 primeiros meses. Escreva o total ganho com juros no período.

#include <stdio.h>

int main()
{
    float montante, taxaJuros;
    printf("Digite o depósito inicial: ");
    scanf("%f", &montante);
    printf("Agora informe a taxa de juros (%): ");
    scanf("%f", &taxaJuros);
    taxaJuros /= 100;
    char meses[12][20] = {"Janeiro", "Fevereiro", "Março", "Abril", "Maio", "Junho", "Julho", "Agosto", "Setembro", "Outubro", "Novembro", "Dezembro"};
    for (int i = 0; i < 24; i++)
    {
        printf("%s ", meses[i % 12]);
        printf("ano %d ", (i / 12) + 1);
        printf(": R$%.2f\n", montante *= (1 + taxaJuros));
    }
    return 0;
}